pmoves/services/agentgym-rl-coordinator/app.py

```
@@ -1,68 +1,356 @@
+
+@app.get("/agentgym/trajectories/{trajectory_id}")
+async def get_trajectory(trajectory_id: str):
+    """Get a specific trajectory by ID."""
+    # Validate trajectory_id is a UUID
+    try:
+        UUID(trajectory_id)
+    except ValueError:
+        raise HTTPException(
+            status_code=400,
+            detail="Invalid trajectory_id format. Must be a valid UUID."
+        )
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)